src/app/vesselInfo/vesselInfoActions.js

```diff
@@ -206,13 +215,13 @@ export function hideVesselsInfoPanel() {
   }
 }
 
-export function addVessel({ tilesetId, seriesgroup, fromSearch = false, parentEncounter = null }) {
```

Comment:
👏 i like not couple the addVessel method and have another one to do something different